lib/ProductOpener/Display.pm

```diff
@@ -3898,6 +3898,14 @@ HTML
 						$user_template_data_ref->{edit_profile} = 1;
 						$user_template_data_ref->{orgid} = $orgid;
 					}
+					if (defined $User{pro_moderator}) {
+						my @org_members;
+						foreach my $member_id (sort keys %{$user_or_org_ref->{members}}) {
+							my $member_details = retrieve_user($member_id);
```

Comment:
As a convention, when a variable is an object reference, we suffix it with _ref. Coul you rename $member_details to $member_user_ref or similar?